Day-11-Quiz-App/quizBuilder.js

```diff
@@ -145,7 +144,37 @@ class QuizBuilder {
         const choicesHeader = document.createElement("text");
         const answerHeader = document.createElement("text");
 
+        this._questionsContainer.appendChild(createHR);
         this._questionsContainer.appendChild(questionTextHeader);
+        
+        // Here starts the delete Button
+        if (this._pressedAddQuestionButtonCounter != 0) {
+            const removeQuestionButton = document.createElement("button");
+            this._questionsContainer.appendChild(removeQuestionButton);
+            removeQuestionButton.textContent = "X";
+            removeQuestionButton.className = "removeQuestionButton";
+            choicesHeader.style.display = "inline-block";
+                removeQuestionButton.addEventListener("click", (event) => {
```

Comment:
The `addEventListener` statement is intended one level to far in this case.
Only the function body should be indented.